pkg/plugins/resources/memory/store.go

```diff
@@ -164,6 +182,20 @@ func (c *memoryStore) Delete(_ context.Context, r model.Resource, fs ...store.De
 	if record == nil {
 		return store.ErrorResourceNotFound(r.GetType(), opts.Name, opts.Mesh)
 	}
+	for _, child := range record.Children {
+		_, childRecord := c.findRecord(child.ResourceType, child.Name, child.Mesh)
+		if childRecord == nil {
+			return store.ErrorResourceNotFound(model.ResourceType(child.ResourceType), child.Name, child.Mesh)
+		}
+		obj, err := registry.Global().NewObject(model.ResourceType(child.ResourceType))
+		if err != nil {
+			return fmt.Errorf("MemoryStore.Delete() couldn't delete linked child resource")
+		}
+		_ = c.unmarshalRecord(childRecord, obj)
```

Comment:
Why the error is ignored here and in 186?

Reply:
I guess the logic here was the following - we marshal and unmarshal records only in that file, records are private and owners couldn't be updated. But I don't mind, more error checking is better.
Sorry, I don't see ignoring error in 186 and at that file at all